packages/test/src/load/worker.ts

```diff
@@ -1,11 +1,11 @@
-import arg from 'arg';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-grpc';
 import * as opentelemetry from '@opentelemetry/sdk-node';
 import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions';
-import { Core, Worker, DefaultLogger } from '@temporalio/worker';
-import { CollectorTraceExporter } from '@opentelemetry/exporter-collector-grpc';
```

Comment:
How much are these APIs moving? Or are we just switching to a different one? (otel feels as unstable as it did two years ago)

Reply:
This was a breaking change. There were a number of them in the last 2 versions. 

https://github.com/open-telemetry/opentelemetry-js/blob/main/CHANGELOG.md#101--experimental-0270

I didn't change `packages/*` versions, so don't think should be breaking for our users.